schemas/org.gnome.shell.extensions.gtile.gschema.xml

```diff
@@ -5,6 +5,10 @@
 			<default><![CDATA[['<Super>KP_Enter']]]></default>
 			<summary>The key you want to activate gTile.</summary>
 		</key>
+		<key type="as" name="show-toggle-tiling-alt">
```

Comment:
For some reason this binding doesn't actually work on my machine (Ubuntu 19.04/Gnome 3.32).

It's very strange because the code looks fine for it.